examples/with-claims/api/index.ts

```diff
@@ -0,0 +1,197 @@
+                        const res = await oI.signUp(input);
+                        if (res.status === "OK") {
+                            if (res.user.email.includes("admin")) {
+                                roleDB.set(res.user.id, ["admin"]);
+                            } else {
+                                roleDB.set(res.user.id, ["user"]);
+                            }
+                        }
+                        return res;
+                    },
+                }),
+            },
+        }),
+        Session.init({
+            claimsToAddOnCreation: [RolesClaim, EmailVerifiedClaim],
+            defaultValidatorsForVerification: [EmailVerifiedClaim.isVerified],
+        }),
+    ],
+});
+
+const app = express();
+app.use(
+    cors({
+        origin: websiteDomain, // TODO: Change to your app's website domain
+        allowedHeaders: ["content-type", ...supertokens.getAllCORSHeaders()],
+        methods: ["GET", "PUT", "POST", "DELETE"],
+        credentials: true,
+    })
+);
+
+app.use(
+    helmet({
+        contentSecurityPolicy: false,
+    })
+);
+app.use(morgan("dev") as any);
+app.use(bodyParser.json());
+
+app.use(middleware());
+
+app.get(
+    "/api1",
+    verifySession({
+        overwriteDefaultValidators: [
+            RolesClaim.hasRole.including("admin"),
+            EmailVerifiedClaim.isVerified, // Maybe this could be just EmailVerifiedClaim
+            SecondFactorCheckers.any2Factors,
+        ],
+    }),
+    async (req, res) => {
+        const session: SessionContainer = (req as any).session;
+
+        const roles = RolesClaim.getValueFromPayload(session.getAccessTokenPayload(), {});
+        res.send({
+            status: "OK",
+            roles,
+            mfaOTP: SecondFactorOTPClaim.getValueFromPayload(session.getAccessTokenPayload(), {}),
+        });
+    }
+);
+
+app.get(
+    "/api2",
+    verifySession(), // This only requires the default EmailVerifiedClaim
+    async (req, res) => {
+        const session: SessionContainer = (req as any).session;
+
+        const roles = RolesClaim.getValueFromPayload(session.getAccessTokenPayload(), {});
+        res.send({
+            status: "OK",
+            roles,
+        });
+    }
+);
+
+app.post("/second-factor", verifySession(), async (req, res) => {
+    const session: SessionContainer = (req as any).session;
+    await SecondFactorOTPClaim.addToSession(session, true);
+    res.send({
+        status: "OK",
+    });
+});
+
+app.post(
+    "/refresh-roles",
+    verifySession(), // This only requires the default EmailVerifiedClaim
+    async (req, res) => {
+        const session: SessionContainer = (req as any).session;
+
+        await RolesClaim.updateRoles(session);
+
+        res.send({
+            status: "OK",
+        });
+    }
+);
+
+app.post(
+    "/refresh-email-verified",
+    verifySession({ overwriteDefaultValidators: [] }), // This only requires the default EmailVerifiedClaim
```

Comment:
This is misleading that even if you are providing an empty array, we are still applying email verification claim?